mmv1/products/networksecurity/FirewallEndpointAssociation.yaml

```diff
@@ -93,6 +93,14 @@ properties:
     name: 'labels'
     description: |
       A map of key/value label pairs to assign to the resource.
+  - !ruby/object:Api::Type::Boolean
+    name: 'disabled'
+    default_value: false
```

Comment:
Note: determine if this is backwards compatible or not

Reply:
The test I added should verify the backwards compatibility of the new field.

Reply:
I don't think this resource could work before, so BC can't matter.